### PR TITLE
Frame from video

### PR DESCRIPTION
**Pull request by chooco**
Adding script to convert video to frames, you can add your sample video and it should work

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.idea/workspace.xml
+FILE170318-185831.MP4
```
Comment (jifwin) [inline .gitignore]: Please add this video to repository, so that everyone could test the script.

File: video-to-frames.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import cv2
+vidcap = cv2.VideoCapture('FILE170318-185831.mp4')
```
Comment (jifwin) [inline video-to-frames.py]: [minor] For readability let's stick to full names and "lowercase_separated_by_underscores" naming convention. In this example that would be "video_capture".

File: .gitignore (hunk 1 above)
Comment (chooco) [inline .gitignore]: I add cropped video, please chceck that is working

Comment (jifwin) [inline .gitignore]: Please add cv2 to requirements.txt

Comment (chooco) [inline .gitignore]: Added

**Pull request merged**